docs/index.html

```diff
@@ -34,34 +34,34 @@
     </button>
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
       <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+        
+          
+          
```

Comment:
What's with all the whitespace changes?

Reply:
I'm not too sure, the line numbers seem to the same for both the removed and added